LibGit2/Functions/Test-GitTag.ps1

```diff
@@ -0,0 +1,45 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function Test-GitTag{
+    <#
+    .SYNOPSIS
+    Tests if a tag exists.
+
+    .DESCRIPTION
+    If a tag exists, returns `$true`, otherwise `$false`.  If checking for a tag in a place that doesn't have a directory, you'll get an error and nothing is returned.

```

Comment:
Prefix descritpion with, "The `Test-GitTag function tests if a tag exists in a Git repository." 

What does "If checking for a tag in a place that doesn't have a directory, you'll get an error and nothing is returned." mean? 

Add a sentence about passing the name of the tag to the `Name` parameter after the true/false sentene.